benchmarks/asv.conf.json

```diff
@@ -121,6 +121,7 @@
             // Note: these don't have a minimum in setup.py
             "pytables": "3.6.1",
             "ephem": "3.7.6.0",
+            "numba": "",
```

Comment:
Should we keep all the packages in the "minimum" list pinned to something, even if we don't actually have a minimum?  That way any changes to numba that affect our performance will show up in one env but not the other.

Side note:  I see that [spa.py](https://github.com/pvlib/pvlib-python/blob/master/pvlib/spa.py#L33) mentions a minimum numba version that isn't reflected in [setup.py](https://github.com/pvlib/pvlib-python/blob/master/setup.py#L48).

Reply:
That minimum version is still accurate, as far as I know, but I'd be surprised if it's compatible with the rest of our minimum requirements.

`conda search -c main numba | grep py36` shows that numba 0.36.1 is the oldest package compatible with python 3.6 and numpy 1.12. The numba git history says 0.36.1 was released on Dec 7, 2017. (0.17.0 was released Feb 3, 2015.) I'll add that to the asv conf.

I'd be fine with also adding a minimum numba requirement to setup.py, but I think that should be done in combination with changes to the import logic in spa.py. In particular, I don't like the numpy fallback - that led to hard to track down errors when developing this. It would also be worth reviewing modern numba best practices. If we're adding a minimum numba requirement to setup.py then we should probably be testing against it too. More than I want to tackle in this PR.

Reply:
https://github.com/pvlib/pvlib-python/issues/1060